src/pages/composites/categories/index.tsx

```diff
@@ -1,12 +1,18 @@
-import Categories from '@components/Categories';
```

Comment:
Did you check these removed codes for dead codes, like delete them if they aren't used anywhere else

Reply:
What dead codes are you referring to?

Reply:
[Dead codes](https://en.wikipedia.org/wiki/Dead_code): Codes that are not bein used, imported anywhere. For example [here](https://github.com/Ozoemenamsolomon/orthoex-store/pull/14/files#diff-c19bdcd7a2f8362a946e012e50fc63bd1701a4a73b57890b45ed1cd6ca217c13L18). Check if the `ViewMoreLink` is used or imported anywhere else, if not, delete the file.